metadata/transportation/airport_locations.md

```diff
@@ -0,0 +1,76 @@
+# Title
+
+Utah Airport Locations
+
+## ID
+
+ae01b2d088f64c0fb989c2592e7e4f89
+
+## Brief Summary
+
+Points dataset of airport and heliport locations in Utah.
+
+## Summary
+
+Utah Airport Locations is a subset of the Federal Aviation Administration's National Airspace System Aeronautical Data products. This dataset only includes airports and heliports found in Utah.
+
+## Description
+
+### What is the dataset?
+
+This dataset contains point locations of aircraft landing facilities in the state of Utah, including commercial airports, hospital heliports, and other facilities. This dataset includes attribute data on the physical and operational characteristics of each facility.
+
+### What is the purpose of the dataset?
+
+This dataset has been made available to the public for general analytic and cartographic purposes.
+
+<!--- Is this dataset used in airport information lookups or anything else like that? Does it serve a more specific purpose? --->
+
+### What does the dataset represent?
+
+Each point on this dataset represents the location of an airport or heliport in Utah. Points were placed on or near the main structure of the airport. The lan_fa_ty field indicates the type of landing facility. Other attributes in this dataset include [enplanement](https://www.faa.gov/airports/planning_capacity/passenger_allcargo_stats/passenger) numbers, usage statistics, and a unique facility ID number (locid).
```

Comment:
Maybe something like this:
Points were placed on or near the main structure of the airport, runway, or helipad.

Reply:
Got that added in! Do we have any more information about how this dataset was created? The original metadata refers to this "National Airspace System Resource Aeronautical Data Product", but I'm not sure that's the actual name of the database where this dataset came from, as I couldn't find it anywhere. They have a data download page of aeronautical data products, but none by this name specifically. 

Also, do we know how often this dataset is updated/who maintains it?

Reply:
I'm not sure where it came from, I would have guessed the FAA but it could be from anywhere. I've moved/deleted the occasional point when someone like Blue Stakes lets me know of a change but that's about it.